src/components/MAPI/apps/ClusterDetailAppListWidgetVersionInspector.tsx

```diff
@@ -116,8 +116,12 @@ const ClusterDetailAppListWidgetVersionInspector: React.FC<IClusterDetailAppList
 
   useEffect(() => {
     // Select the current app version when everything is loaded.
-    if (!isLoading && typeof currentSelectedVersion === 'undefined') {
-      const entry = appCatalogEntryList!.items.find(
```

Comment:
`appCatalogEntryList` could've still been `undefined` when `appCatalogEntryListError` would not be `undefined`